get_kg_query_params.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+This script is meant to use network centrality measures to identify and select
+nodes and edges (entities and relations) that would make good prediction
+starting points for the MULTIVAC system. It does this by using eigenvector
+centrality but can be extended to include additional network centrality
+measures.
+"""
+import argparse
+import networkx as nx
+import sys
+
+
+def analyze_network(net, args_dict):
+    if 'degree' in args_dict['measure']:
+        ans = nx.degree_centrality(net)
+    elif 'eigenvector' in args_dict['measure']:
+        ans = nx.eigenvector_centrality(net)
+    else:
+        sys.exit('Whoops; you must provide a valid network centrality measure.')
+    ans = sorted(ans.items(), key=lambda x: x[1])
+    ans.reverse()
```

Comment:
note: you could consolidate lines 22-23 into:
```python
ans = sorted(ans.items(), key=lambda x: x[1], reverse=True)
```

Reply:
Consider it done...